text/0000-const-trait-impls.md

```diff
@@ -15,21 +15,21 @@ Make trait methods callable in const contexts. This includes the following parts
 Fully contained example ([Playground of currently working example](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2021&gist=2ab8d572c63bcf116b93c632705ddc1b)):
 
 ```rust
-const trait Default {
-    (const) fn default() -> Self;
+[const] trait Default {
```

Comment:
the jury is still out on this one, as similarly to `const fn` there is no benefit of choosing `[const] trait` over `const trait`, as the latter can't mean anything different

Reply:
Yeah, that's also true